### PR TITLE
Post driver

### PR DESCRIPTION
**Pull request by kbhar01**
Post Drivers - also merged from main

### CODE REVIEW

File: src/test/java/com/galvanize/speedway/integrationtest/DriverIT.java [hunk 1]
```diff
@@ -30,15 +37,41 @@ public void getDriversTest() throws Exception{
...
+
+      mockMvc.perform(rq)
+              .andExpect(status().isCreated())
+              .andDo(print());
+
+      this.mockMvc.perform(get("/speedway/drivers"))
+              .andExpect(status().isOk())
+              .andExpect(jsonPath("length()").value(1))
+         .andDo(document("GetDrivers",responseFields(
+            fieldWithPath("[0].firstName").description("First Name of driver"),
```
Comment (davidgrieser) [inline src/test/java/com/galvanize/speedway/integrationtest/DriverIT.java:67]: Nice REST docs!

**Review by Christopher-Diehl-cognizant-com — DISMISSED** [review]

File: src/test/java/com/galvanize/speedway/integrationtest/DriverIT.java [hunk 2]
```diff
@@ -30,15 +37,41 @@ public void getDriversTest() throws Exception{
...
+      driver.setNickName("Test Nickname");
+
+
+      RequestBuilder rq = post("/speedway/drivers")
+              .content(objectMapper.writeValueAsString(driver))
+              .contentType(MediaType.APPLICATION_JSON);
+
+      mockMvc.perform(rq)
+              .andExpect(status().isCreated())
+              .andDo(print());
```
Comment (davidgrieser) [inline src/test/java/com/galvanize/speedway/integrationtest/DriverIT.java:61]: Did you want to document how `POST`s should be formatted?

**Review by davidgrieser — DISMISSED** [review]
Nice catch on winds vs wins. Good addition of `POST`, but it isn't documented like `GET` currently is.

File: src/test/java/com/galvanize/speedway/integrationtest/DriverIT.java (hunk 2 above)
Comment (kbhar01) [reply src/test/java/com/galvanize/speedway/integrationtest/DriverIT.java:61]: I will add it

**Comment by sebinfrancis** [comment]
Have you added unit test for Driver Service?

**Review by Christopher-Diehl-cognizant-com — APPROVED** [review]

**Review by sebinfrancis — CHANGES_REQUESTED** [review]
for addDriver do we need an unit test

**Review by Christopher-Diehl-cognizant-com — APPROVED** [review]

**Review by sebinfrancis — APPROVED** [review]

**Pull request merged**